bucket/yaffplayer.json

```diff
@@ -0,0 +1,39 @@
+{
+    "version": "0.5.7.0",
+    "description": "A simple video player based on FFmpeg libraries",
+    "homepage": "https://www.geeks3d.com/yaffplayer/",
+    "license": "Freeware",
+    "architecture": {
+        "64bit": {
+            "url": "https://www.geeks3d.com/dl/get/618#/dl.zip",
+            "hash": "md5:c7d09f08837dc3d49891e317fc8e94bd"
+        }
+    },
+    "extract_dir": "YAFFplayer",
+    "shortcuts": [
+        [
+            "YAFFplayer.exe",
+            "YAFFplayer"
+        ]
+    ],
+    "persist": [
+        "_geexlab_log.txt",
+        "_scene_init_log.txt",
+        "imgui.ini"
+    ],
+    "checkver": {
+        "url": "https://www.geeks3d.com/dl/show/618",
+        "regex": "Version:.*?([\\d.]+)"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://www.geeks3d.com/dl/get/618#/dl.zip",
```

Comment:
I have a slightly unrelated question (manifest is good to go, nothing to change): Is the URL field necessary here? Because hash is changing, so we have a hash field, but URL is static.

Reply:
Good idea. I Never thought about that.

Unfortunately *autoupdate* cannot work properly without the `autoupdate.url` field (the hash field will be empty after running *autoupdate*), we may need to investigate that later.

Reply:
Okay, thanks for the info!